src/main/java/MisterDuke.java

```diff
@@ -0,0 +1,678 @@
+import java.io.*;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Locale;
+import java.util.Scanner;
+
+/**
+ * Mister Duke is an interactive chat bot that
+ * keep track of your tasks, deadlines and events
+ * in a list.
+ * Some things Mister Duke is good at:
+ * - listing the tasks on your list
+ * - marking specified tasks as done
+ * - removing specified tasks from the list
+ * - find matching tasks given a search word/phrase
+ *
+ * @author Shaelyn
+ * @version CS2103T 20/21 Semester 2, Individual Project
+ */
+
+public class MisterDuke {
+
+    private final Ui ui;
+    private final Storage storage;
+    private ArrayList<Task> tasks;
+
+    public MisterDuke(String filePath) throws IOException {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new ArrayList<>(storage.load());
+        } catch (IOException e) {
+            tasks = new ArrayList<>();
+        }
+    }
+
+    /**
+     * Runs Mister Duke
+     * @throws IOException when the input is wrong/incomplete
```

Comment:
Should there be a line break between description and parameters section? Can also look at other Javadocs.